launcher/agent/agent.go

```diff
@@ -114,6 +117,11 @@ func (a *agent) Attest(ctx context.Context, opts AttestAgentOpts) ([]byte, error
 		},
 	}
 
+	if a.cloudLogger != nil {
+		a.cloudLogger.Log(logging.Entry{Payload: challenge})
```

Comment:
As I commented on #375, using the agent is a bit of an odd choice since the agent is closely tied with the launcher (consider `MeasureEvent`). We should really refactor out the Attest command if it is useful to you.

Reply:
The refactor out would be in another PR. I think we might want to discuss on where to host verifier package and agent package. I would PM you later!